package.json

```diff
@@ -15,7 +15,7 @@
     "lint:prettier": "prettier --check \"{,!(node_modules|dist|static|public)/**/}*.{js,ts,cjs,mjs,json}\" --ignore-path .gitignore",
     "lint:prettier:fix": "prettier --write \"{,!(node_modules|dist|static|public)/**/}*.{js,ts,cjs,mjs,json}\" --ignore-path .gitignore",
     "docker": "docker build . && docker run --rm -i -p 8080:8080 $(docker build -q .)",
-    "prepare": "node -e \"if (process.env.NODE_ENV !== 'production'){ process.exit(1) } \" || husky install"
+    "prepare": "is-ci || husky install"
```

Comment:
using [`is-ci`](https://github.com/watson/is-ci) implementation as per [discord.js](https://github.com/discordjs/discord.js/blob/main/package.json#L22)

Reply:
isn't "husky install" depreciated now?